babel.config.js

```diff
@@ -4,6 +4,8 @@ module.exports = {
       "@babel/preset-env",
       {
         targets: { node: 14 },
+        // necessary to preserve dynamic import of ESM-only modules
+        exclude: ["proposal-dynamic-import"],
```

Comment:
have you checked the build output? i'd expect that preconstruct flag I mentioned to be required too - although I never really used it myself so far

Reply:
The preconstruct flag was already set, and the binary worked correctly with `yarn preconstruct build` but not with `yarn preconstruct dev` until the babel change was made.

Reply:
ah cool, I see we are using it since https://github.com/Thinkmill/manypkg/pull/181

Reply:
I wonder if this would still be required if we'd update all of the Babel deps. Maybe we just have an outdated "stable features" list in the `@babel/preset-env` since we are using an old version?